### PR TITLE
Add required files

### PR DESCRIPTION
**Pull request by raezon**
We did create a hello  microverse  i mention the new inserted files

- [x] index.html
- [x] styles.css
- [x] .gitignore(ignore node_modules and test.md)
- [x] .hintrc
- [x] .stylelintrc.json
- [x] linters.yml

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse</h1>
```
Comment (Abenezer-Tilahun) [inline index.html:11]: - [ ] As stated the project requirement I recommend you to add A .css file that includes `one class` with styles for your header.
Capture

Comment (Abenezer-Tilahun) [inline index.html]: - [x] As stated the project requirement kindly make sure that you added A .css file that includes `one class` with styles for your header.
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @raezon    , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional  README 
✔️ Good commit messages

almost

## Required Changes ♻️

- [x] As stated the project requirement in `.gitignore` file make sure you add A test.md file with the text "This file should be ignored by git" is actually ignored by git and is not present in your GitHub repository.

https://github.com/raezon/Hello-Microverse/blob/472db85ae1a9b37f35d1ecb22e1b4c00571ea208/.gitignore#L1-L104


_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (raezon) [reply index.html]: okey

Comment (raezon) [reply index.html]: done

**Review by Santiago220991 — APPROVED** [review]
Hi @raezon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**